Changed README.md (#2)
PashaTerletskyi/taxi-service: README.md

# **TAXI SERVICE**

> Simple website. where you can make connection taxi drivers with cars.

Reviewer: ```suggestion
Simple website, where you can connect taxi drivers with cars.
```

## _FUNCTIONS :_

1. Create new manufacturer.
1. Delete manufacturer.(Soft delete)
1. Display all manufacturers.   
1. Create new driver.
1. Delete driver.(Soft delete)
1. Display all drivers.
1. Create new car.
1. Delete car.(Soft delete)
1. Display all cars.
1. Display all car with drivers.
1. Display cars for current driver.
1. Add connections cars to drivers.

## _PROJECT USES :_ 

* Java 11.
* Maven 4.0.0.
* Servlet API 4.0.1.
* JSTL 1.2.
* Log4j 2.14.1.
* Mysql Connector Java 8.0.22. 
* Maven Checkstyle Plugin 3.1.1.
* Tomcat 9.0.50.
* MySQL Workbench 8.0.25.

## _DATABASE STRUCTURE :_

<img src="https://i.ibb.co/9ZbqZHN/1.png" alt="1">

## _HOW TO RUN PROJECT :_

> 1. Create new maven project in your IDE.

Reviewer: ```suggestion
1. Clone this project and open it in your IDEA.
```
